src/components/React/Howto.js

```diff
@@ -29,117 +29,132 @@ export default function Howto(props) {
           <h2 id="user-guide" className="mb-4">
             User Guide
           </h2>
-          <Row>
-            <p>
-              Welcome to the Web-VR Volume viewer. This short guide is meant to
-              help new users on to use the application and and showcase the
-              potential its potential for future research.
-            </p>
-            <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/guide12.png"
-              alt="Example screenshot"
-              width="100%"
-            />
+          <Row class>
```

Comment:
The class tag doesn't do anything
```suggestion
          <Row>
```

Reply:
@RobertGemmaJr   help me with the CSS changes, and push them in this PR :+1: